extensions/stackdriver/common/utils.h

```diff
@@ -42,6 +43,13 @@ void buildEnvoyGrpcService(
     const StackdriverStubOption &option,
     ::envoy::config::core::v3::GrpcService *grpc_service);
 
+// Return Owner information when that information. If that information
```

Comment:
nit: comment seems to be incomplete sentence.

Reply:
fixed.